src/node/Main.hs

```diff
@@ -73,7 +73,10 @@ action args@SimpleNodeArgs {..} = do
         sscParams = bool (Left ()) (Right gtParams) (CLI.sscAlgo commonArgs == GodTossingAlgo)
     case (sscParams) of
         (Left par)  -> actionWithoutWallet @SscNistBeacon par currentParams
-        (Right par) -> actionWithoutWallet @SscGodTossing par currentParams
+        (Right par) ->
+            -- TODO: What the heck?!
+            -- @actionWithoutWallet@ requires HasCoreConstants, why?
+            giveStaticConsts (actionWithoutWallet @SscGodTossing par currentParams)
```

Comment:
seems it's false, I see error
```
    /home/pva701/github/cardano-sl/src/Pos/Launcher/Launcher.hs:35:32: error:
        • Could not deduce (Data.Reflection.Given
                              Pos.Core.Context.CoreConstants)
```
where constraints are `(SscConstraint ssc, SecurityWorkersClass ssc)`